pkgs/by-name/od/odamex/package.nix

```
@@ -5,32 +5,52 @@
   cmake,
   pkg-config,
   makeWrapper,
-  SDL,
-  SDL_mixer,
-  SDL_net,
+  SDL2,
+  SDL2_mixer,
+  SDL2_net,
   wxGTK32,
+  zlib,
+  fltk,
+  curl,
+  cairo,
+  pango,
+  glfw,
+  glm,
+  alsa-lib,
+  deutex,
+  portmidi,
 }:
 
 stdenv.mkDerivation rec {
   pname = "odamex";
-  version = "0.9.5";
+  version = "11.0.0";
 
   src = fetchurl {
-    url = "mirror://sourceforge/${pname}/${pname}-src-${version}.tar.bz2";
-    sha256 = "sha256-WBqO5fWzemw1kYlY192v0nnZkbIEVuWmjWYMy+1ODPQ=";
+    url = "mirror://sourceforge/${pname}/${pname}-src-${version}.tar.gz";
+    sha256 = "7e4e83ac08546b778ea9e08d5a34a82a0f57f3505bde3ad4aba265a13c1f1387";
```

```suggestion
    hash = "";
```
Now that you're changing this line, consider
1. changing the attr to `hash` and
2. regenerate the hash itself with Nix (before submitting!) (try to build the package (preferably with `nix-build -A odamex`) and get the correct hash from the error) -- such format is not acceptable in Nixpkgs